smac/facade/roar_facade.py

```diff
@@ -8,7 +8,8 @@
 from smac.tae.execute_ta_run import StatusType, ExecuteTARun
 from smac.stats.stats import Stats
 from smac.scenario.scenario import Scenario
-from smac.optimizer.ei_optimization import ChooserNoCoolDown, ChooserLinearCoolDown
+from smac.optimizer.random_configuration_chooser import ChooserNoCoolDown, \
```

Comment:
Sorry, I can't comment at the correct location, but what does the random_configuration_chooser do for ROAR? Is it really needed.

Reply:
Removed it from ROAR